verification/traits/src/lib.rs

```diff
@@ -59,42 +39,38 @@ impl Debug for dyn VerificationStep {
 /// This is will be a short circuiting operation. If the `left` side fails
 /// the `right` side will *not* be exercised.
 #[derive(Debug)]
-pub struct And {
-    left: Box<dyn VerificationStep>,
-    right: Box<dyn VerificationStep>,
+pub struct And<L, R> {
```

Comment:
I know convention is often `T` (type) and then `U`, but this seemed like a case where `L` for left and `R` for right was more intuitive.